.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,63 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fluent UI React-Native? This template includes necessary information to get started, and steps to completion
+---
+
+<!-- Use this template for new components or new component variants -->
+
+## Component details
+
+### Description
+
+<!-- fill this out -->
+
+### Which product teams/scenarios need this control
+
+<!-- The more teams/scenarios that would use this control the better chance it will get prioritized -->
+
+### Design assets
+
+<!-- Please provide links to redlines or screenshots of intended component design -->
+<!-- Are you internal to Microsoft and your design isn't yet public? Please reach out to us internally via the Fluent UI Apple Teams channel or log a request at https://aka.ms/fluentui-native. -->
+
+### Component ownership
+
+<!-- Are there one or more people who can help maintain this component over time? Who will address bugs? -->
+
+## Component composition
+
+### Imports/Dependencies
+
+<!-- What other dependencies will your component be using -->
+
+### Proposed Public API
+
+<!-- Define the proposed public API for this control -->
+
+### Intended Platforms
+- [ ] iOS
+- [ ] macOS
+- [ ] win32
+- [ ] windows
+
+## Component code
+
+### Example code
+
+```Swift
+```
+
+### New Component Checklist
```

Comment:
This seems highly Apple biased (light and dark appearance, different resolutions, size classes and window sizes, "VoiceOver", etc. Maybe we should make this split by platforms?

Reply:
The templates took on an apple skew due to what I know but I can split it up over platforms. Will probably need win/android engineers to contribute to their platforms in that case